app-backup/deja-dup/deja-dup-42.7.ebuild

```diff
@@ -0,0 +1,56 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit meson eutils gnome2-utils vala xdg
+
+DESCRIPTION="Simple backup tool using duplicity back-end"
+HOMEPAGE="https://gitlab.gnome.org/World/deja-dup"
+SRC_URI="https://gitlab.gnome.org/World/${PN}/-/archive/${PV}/${P}.tar.bz2"
+
+LICENSE="GPL-3"
+SLOT="0"
+KEYWORDS="~amd64 ~x86 ~amd64-linux ~x86-linux"
```

Comment:
Please don't add prefix keywords to new ebuilds. Did you test on x86?

Reply:
No, this is a mistake. I just added those as my ebuild is based on the older ebuild.